src/components/Editor/EmptyLines.css

```diff
@@ -1,7 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
-
+/*
```

Comment:
I believe we still want this, right?

Reply:
oh... lets rm this file

Reply:
Sorry!  I wasn't aware -- remove away! :)